lib/faker/address.ex

```diff
@@ -9,146 +9,1232 @@ defmodule Faker.Address do
 
   @doc """
   Return random building number.
+
+  ## Examples
+
+      iex> Faker.Address.building_number()
+      #=> "99"
   """
-  @spec building_number() :: String.t
+  @spec building_number() :: String.t()
   def building_number do
     ["#####", "####", "###", "##", "#"]
     |> Enum.at(Faker.random_between(0, 4))
-    |> Faker.format
+    |> Faker.format()
   end
 
   @doc """
   Return city name.
+
+  ## Examples
+
+      iex> Faker.Address.city()
+      #=> "Palmaberg"
   """
-  @spec city() :: String.t
+  @spec city() :: String.t()
   def city do
     city(Faker.random_between(0, 3))
   end
 
-  defp city(0), do: "#{city_prefix()} #{Name.first_name}#{city_suffix()}"
-  defp city(1), do: "#{city_prefix()} #{Name.first_name}"
-  defp city(2), do: "#{Name.first_name}#{city_suffix()}"
-  defp city(3), do: "#{Name.last_name}#{city_suffix()}"
+  defp city(0), do: "#{city_prefix()} #{Name.first_name()}#{city_suffix()}"
+  defp city(1), do: "#{city_prefix()} #{Name.first_name()}"
+  defp city(2), do: "#{Name.first_name()}#{city_suffix()}"
+  defp city(3), do: "#{Name.last_name()}#{city_suffix()}"
 
   @doc """
   Return city prefix.
+
+  ## Examples
+
+      iex> Faker.address.city_prefix()
+      #=> "West"
   """
-  @spec city_prefix() :: String.t
-  sampler :city_prefix, ["North", "East", "West", "South", "New", "Lake", "Port"]
+  @spec city_prefix() :: String.t()
+  sampler(:city_prefix, ["North", "East", "West", "South", "New", "Lake", "Port"])
 
   @doc """
   Return city suffix.
+
+  ## Examples
+
+      iex> Faker.Address.city_suffix()
+      #=> "ton"
   """
-  @spec city_suffix() :: String.t
-  sampler :city_suffix, ["town", "ton", "land", "ville", "berg", "burgh", "borough", "bury", "view", "port", "mouth", "stad", "furt", "chester", "mouth", "fort", "haven", "side", "shire"]
+  @spec city_suffix() :: String.t()
+  sampler(:city_suffix, [
+    "town",
+    "ton",
+    "land",
+    "ville",
+    "berg",
+    "burgh",
+    "borough",
+    "bury",
+    "view",
+    "port",
+    "mouth",
+    "stad",
+    "furt",
+    "chester",
+    "mouth",
+    "fort",
+    "haven",
+    "side",
+    "shire"
+  ])
```

Comment:
I just noticed these now: Why did you change the format? I get that some people might prefer it this way, but the changes are quite major and not relevant to the PR. If I had to review this thoroughly, I actually should check if you didn't change the data by accident.